cmd-record.c

```diff
@@ -1266,7 +1267,13 @@ static void print_child_time(struct timespec *ts1, struct timespec *ts2)
 		sec--;
 	}
 
-	pr_out("elapsed time: %"PRIu64".%09"PRIu64" sec\n", sec, nsec);
+	sprintf(buf, "%"PRIu64".%09"PRIu64" sec", sec, nsec);
+	return xstrdup(buf);
+}
```

Comment:
It's unnatural to keep it as a string but I think it's ok for this case since we need to do the same job twice.  And then please replace snprint + xstdup into xasprint.

Reply:
I got it.